src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,85 @@
-const Countdown = () => null;
+import { useState } from 'react';
+import { PlayPauseButton, FastForwardButton, ResetButton } from '../generic/Button';
+import { Input } from '../generic/Input';
+import { DisplayTime } from '../generic/DisplayTime';
+import { convertToMs } from '../../utils/helpers';
+
+const Countdown = () => {
+    const [countdownMinValue, setCountdownMinValue] = useState(0);
+    const [countdownSecValue, setCountdownSecValue] = useState(0);
+    const [countdownTime, setCountdownTime] = useState(0);
+    const [isCountdownRunning, setIsCountdownRunning] = useState(false);
+    const [intervalId, setIntervalId] = useState(0);
+
+    // play/pause Countdown timer
+    const handleStart = () => {
```

Comment:
In this component, you can only clearInterval by a user action - namely, clicking the buttons. This mostly works in the layout of your app currently, but can cause memory leaks down the line if you end up changing your design, and your component may have reason to unmount.

I would recommend adding handling in a useEffect to clear the interval in case of an unmount.